src/config/embeddings.yaml

```
@@ -0,0 +1,58 @@
+# Configuration for embeddings module
+
+jina_v4:
+  # Model settings
+  model_name: "jinaai/jina-embeddings-v4"
+  trust_remote_code: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)